phpBB/includes/acp/acp_email.php

```diff
@@ -225,52 +219,36 @@ function main($id, $mode)
 				for ($i = 0, $size = count($email_list); $i < $size; $i++)
 				{
 					$used_lang = $email_list[$i][0]['lang'];
-					$used_method = $email_list[$i][0]['method'];
 
 					/**
 					 * @var \phpbb\messenger\method\messenger_interface $messenger_method
 					 * @psalm-suppress UndefinedMethod
 					 */
 					foreach ($messenger_collection_iterator as $messenger_method)
```

Comment:
As far as this module is email-dedicated only, this possibly should only use email method to send messages.